bench/src/lib.rs

```diff
@@ -61,19 +61,14 @@ pub async fn connect_client(
         .bind(&SocketAddr::new(IpAddr::V6(Ipv6Addr::LOCALHOST), 0))
         .unwrap();
 
-    let mut config = quinn::ClientConfig::default();
-    let crypto_config = Arc::get_mut(&mut config.crypto).unwrap();
+    let mut client_config =
+        quinn::ClientConfig::with_root_certificates(vec![server_cert.clone()]).unwrap();
```

Comment:
Is the `clone` here really needed? `server_cert` is owned and doesn't seem to be used below.

Reply:
You're right. I'm surprised clippy didn't make a fuss about this one.